app/views/questions/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div id='container'>
+
+  <% @questions.each do |question| %>
+    <div class="question" id='<%= "#{question.id}" %>'>
```

Comment:
We should chat about what you're using this id value for. If you're using it for styling or semantics, you should prefix the numeric id with something meaningful. HTML5 gives us this super neat thing called data attributes. It lets you set `data-question-id="5"` on a div and then reference that elsewhere. This page seems to have a good explanation for you on it, but let me know if you want to chat more after you take a look! https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes